app/services/PensionsService.scala

```diff
@@ -259,6 +261,30 @@ class PensionsServiceImpl @Inject() (reliefsConnector: PensionReliefsConnector,
     } yield ()
   }
 
+  def getShortServiceRefunds(ctx: JourneyContextWithNino)(implicit hc: HeaderCarrier): ApiResultT[Option[ShortServiceRefundsAnswers]] =
+    for {
+      maybeCharges   <- chargesConnector.getPensionChargesT(ctx.nino, ctx.taxYear)
+      maybeDbAnswers <- repository.getAnswers[ShortServiceRefundsStorageAnswers](ctx.toJourneyContext(Journey.ShortServiceRefunds))
+      shortServiceRefundsAnswers = maybeCharges.flatMap(
+        _.overseasPensionContributions.flatMap(
+          _.toShortServiceRefundsAnswers(maybeDbAnswers)
+        ))
+    } yield shortServiceRefundsAnswers
+
+  def upsertShortServiceRefunds(ctx: JourneyContextWithNino, answers: ShortServiceRefundsAnswers)(implicit hc: HeaderCarrier): ApiResultT[Unit] = {
+    val storageAnswers = ShortServiceRefundsStorageAnswers.fromJourneyAnswers(answers)
+    val journeyCtx     = ctx.toJourneyContext(Journey.ShortServiceRefunds)
+
+    for {
+      getCharges <- chargesConnector.getPensionChargesT(ctx.nino, ctx.taxYear)
+      existingCharges = getCharges.map(_.toCreateUpdatePensionChargesRequestModel).getOrElse(CreateUpdatePensionChargesRequestModel.empty)
+      overseasPensionContributions = answers.toOverseasPensions.some
+      updatedCharges               = existingCharges.copy(overseasPensionContributions = overseasPensionContributions)
+      _ <- chargesConnector.createUpdatePensionChargesT(ctx, updatedCharges)
```

Comment:
I think we may need to use createOrDeleteReliefsWhenEmpty method here.

You can test by selecting No for your journeys if it works. Usually these IFS schemas don't let us send an empty JSON, instead we call DELETE if we want to remove previously selected answers and the new object is empty. Have a look on the latest main.